str.length() is not long unsigned int on 32bit arch

While compiling git master for 32bit arch:
```
/moneymanagerex/src/webserver.cpp: In function 'void handle_sql(mg_connection*, http_message*)':
/moneymanagerex/src/webserver.cpp:64:76: warning: format '%lu' expects argument of type 'long unsigned int', but argument 3 has type 'std::__cxx11::basic_string<wchar_t>::size_type {aka unsigned int}' [-Wformat=]
                 "Content-Length: %lu\r\n\r\n%ls", str.length(), str.c_str());
                                                                            ^
```

--- a/src/webserver.cpp
+++ b/src/webserver.cpp
@@ -61,7 +61,7 @@ static void handle_sql(struct mg_connection* nc, struct http_message* hm)
 
     mg_printf(nc, "HTTP/1.1 200 OK\r\n"
                 "Content-Type: application/json; charset=utf-8\r\n"
-                "Content-Length: %lu\r\n\r\n%ls", str.length(), str.c_str());
+                "Content-Length: %zu\r\n\r\n%ls", str.length(), str.c_str());
 }
 
 static void ev_handler(struct mg_connection *nc, int ev, void *ev_data) 

